src/pranitaPJun20/MaxFreqWordString_Collections.java

```diff
@@ -0,0 +1,53 @@
+/*Assignment-37 : 2nd Aug'2020
+
+ * Input : Hi Globant techno Hi Hello Globant techno techno Globant techno
+FR - 1 : freq of each word
+FR - 2 : Maxmimum repeated word and its frequency
+Output :  Hi -> 2
+          Globant -> 3
+          techno -> 4
+          Hello -> 1
+Maximum repeated word is techno having frequency 4.
+*/
+
+package pranitaPJun20;
+
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class MaxFreqWordString_Collections {
+
+	void findMaxRepeatedWordFrequency(String input) {
+		String[] inputArray = input.split(" ");
+		int count = 0;
+		Map<String, Integer> wordMap = new LinkedHashMap<String, Integer>();
+		for (String name : inputArray) {
+			if (wordMap.containsKey(name)) {
+				wordMap.put(name, wordMap.get(name) + 1);
```

Comment:
Hi @PranitaPJun20 
Here you need to write

Line 39 to 42

What you write that is also option but we discuss Try to check with in if condition.

Reply:
Ok Harsh.Will update my code